subprojects/scala/src/main/java/org/gradle/api/tasks/scala/ScalaCompile.java

```diff
@@ -66,6 +69,37 @@ public void setScalaClasspath(FileCollection scalaClasspath) {
         this.scalaClasspath = scalaClasspath;
     }
 
+    /**
+     * Returns the scala compiler plugins to use.
+     *
+     * @since 6.1
```

Comment:
will need to be bumped to 6.3 or 6.4 when PR is ready, same for the other added public method